scss/_common.scss

```
@@ -6,6 +6,7 @@ body {
 }
 
 html {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif;
```

`system-ui` is the standerdized (working Draft) keyword:
```suggestion
  font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif;
```
see https://caniuse.com/#feat=font-family-system-ui

---

I just used what we use in Bootstrap, minus the Emoji fonts.

---

nvm, not sure if https://infinnie.github.io/blog/2017/systemui.html still applies